pinot-minion/src/main/java/org/apache/pinot/minion/MinionConf.java

```diff
@@ -31,6 +31,13 @@ public class MinionConf extends PinotConfiguration {
   public static final String MINION_TASK_PROGRESS_MANAGER_CLASS = "pinot.minion.taskProgressManager.class";
   public static final int DEFAULT_END_REPLACE_SEGMENTS_SOCKET_TIMEOUT_MS = 10 * 60 * 1000; // 10 mins
 
+  /**
+   * The number of threads to use for downloading segments from the deepstore.
+   * This is a global setting that applies to all tasks of BaseMultipleSegmentsConversionExecutor class.
```

Comment:
nit - lets document that this config is applied at a task level and total parallelism will be this factor * number of parallel tasks running.

that will useful to clarify in documentation because this config is at a minion level and can be easily misinterpreted to be a minion level factor

Reply:
It should not be the case. As the default value for `DEFAULT_SEGMENT_DOWNLOAD_PARALLELISM = 1` config is set to 1 and it can not be interpreted that only 1 segment download would happen across the tasks.